chirp/drivers/baofeng_wp970i.py

```diff
@@ -73,7 +73,7 @@ def model_match(cls, data):
 
     if len(data) > 0x2008:
         rid = data[0x2008:0x2010]
-        return rid.startswith(cls.MODEL)
+        return rid.startswith(b'cls.MODEL')
```

Comment:
This is not right. Instead of passing the definition of MODEL on cls, you're passing the bytestring `cls.MODEL`. I assume this is not failing tests because we don't have a legacy test image that requires actual detection. You need to encode `cls.MODEL` to bytes here to properly handle this, so:

`return rid.startswith(cls.MODEL.encode())`

Reply:
Done